tests/unit/src/test/scala/tests/RangeFormattingSuite.scala

```diff
@@ -82,6 +82,27 @@ object RangeFormattingSuite extends BaseSlowSuite("rangeFormatting") {
        |}""".stripMargin
   )
 
+  check(
+    "pasting from one multilineString to another",
```

Comment:
We should keep the names short something like `with-pipes`
This way you can run:
`sbt unit/testOnly -- RangeFormattingSuite.with-pipes`
it's a bit more tricky with longer names.

Reply:
Sounds good. I'll follow that convention!